NotificationHubs/pom.xml

```diff
@@ -3,7 +3,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.windowsazure</groupId>
 	<artifactId>Notification-Hubs-java-sdk</artifactId>
-	<version>0.0.5</version>
+	<version>0.1.0</version>
```

Comment:
No change necessary here. Just wanted to point something out.

Given that there is an actual breaking change to the interface of HttpClientManager, semver would say we need to make this a major version change (e.g., `1.0.0`). However, it seems common practice to not worry so much about semver until a major version is > 0 in the first place.

Reply:
Agree. Above breaking change rule applies only to versions `>= 1.0.0`. In this case the version is "pre-release" so it doesn't have to be marked as `1.0.0`.